internal/gamepad/gamepad_desktop_windows.go

```diff
@@ -612,7 +612,7 @@ func (g *nativeGamepadDesktop) update(gamepads *gamepads) (err error) {
 				return err
 			}
 			// Acquire can return an error just after a gamepad is disconnected. Ignore the error.
-			g.dinputDevice.Acquire()
+			_ = g.dinputDevice.Acquire()
```

Comment:
Handle the error

Reply:
are you sure? the comment seems to indicate we *should* ignore this

Reply:
Oops sorry. That's right...